net-core/README.md

```diff
@@ -17,3 +17,4 @@ At [Wolox](http://wolox.com.ar), we have developed the [NET Core kickoff guide](
 
 - [Creating your first .NET Core project on Linux](https://medium.com/wolox-driving-innovation/how-to-create-your-first-net-e2223dedb74f)
 - [How to create a .NET Core project on Linux using MVC and Postgres](https://medium.com/wolox-driving-innovation/create-an-mvc-web-app-in-net-core-30f92cf94a75)
+- [Migrating from .NET Core 1 to .Net Core 2.0 ](https://medium.com/wolox-driving-innovation/)
```

Comment:
Could it be the actual post is missing? This link redirects to the main Wolox medium page.

Reply:
Yes, I am going to post in medium in the next week, but I needed to do the PR :P

Reply:
LOL just read the [WIP]

Reply:
Jaja oks :P